### DISCUSSION TITLE
User Story

### DISCUSSION THREAD

**Discussion by luisy-eng** [manoa-inspire/MATP#3] [General]
As a user,  I want to have explanations defining financial jargon so that I understand exactly what to input.

**Comment by luisy-eng** [comment]
As a user, I want to view a summary dashboard that displays key financial metrics so that I can quickly assess the financial status.

**Comment by Sydnee-You** [comment]
User story 3 - As a user, I want to be able to view projected impact of the financial state based on proven historical data so that I can make educated decisions about what to do next based on its state.

**Comment by cjochim** [comment]
As a user, I want to be able to navigate and easily identify where to put variables/values so that I would not have to worry about damaging or changing any original data.
